src/code/11_miniMAL.js

```diff
@@ -371,7 +374,7 @@
                             <div class="mdl-card__title mdl-card--border">\
                                 <h2 class="mdl-card__title-text">General</h2>\
                                 </div>';
-                settingsUI += materialCheckbox(autoTracking,'autoTracking','Autotracking');
+                settingsUI += materialCheckbox(autoTracking,'autoTracking','Autotracking'+getTooltip('Autotracking is the function where this script automatically updates the anime´s you watch with your MAL account.','','bottom'));
```

Comment:
I think this should probably be deleted, right?
'',
it is near the end of line 377.